DuckDuckGoTests/Subscription/SubscriptionContainerViewModelTests.swift

```diff
@@ -24,7 +24,21 @@ import SubscriptionTestingUtilities
 
 final class SubscriptionContainerViewModelTests: XCTestCase {
     var sut: SubscriptionContainerViewModel!
-    let subscriptionManager = MockDependencyProvider().subscriptionManager
+
+    let subscriptionManager: SubscriptionManager = {
+        let accountManager = AccountManagerMock()
+        let subscriptionService = DefaultSubscriptionEndpointService(currentServiceEnvironment: .production)
+        let authService = DefaultAuthEndpointService(currentServiceEnvironment: .production)
+        let storePurchaseManager = DefaultStorePurchaseManager()
+        return SubscriptionManagerMock(accountManager: accountManager,
+                                       subscriptionEndpointService: subscriptionService,
+                                       authEndpointService: authService,
+                                       storePurchaseManager: storePurchaseManager,
+                                       currentEnvironment: SubscriptionEnvironment(serviceEnvironment: .production,
+                                                                                   purchasePlatform: .appStore),
+                                       canPurchase: true)
+    }()
```

Comment:
This could probably be done via some function to allow configuring the mocks, but since nothing needs that for now I haven't done it. Let me know if you'd prefer some other approach.